src/pages/Swap/index.tsx

```diff
@@ -650,6 +665,14 @@ export function Swap({
                 showCommonBases
                 id={InterfaceSectionName.CURRENCY_OUTPUT_PANEL}
                 loading={independentField === Field.INPUT && routeIsSyncing}
+                numericalInputSettings={{
+                  // We disable numerical input here if the selected token has tax, since we cannot guarantee exact_outputs for FOT tokens
+                  disabled: outputTokenHasTax,
+                  // Focus the input currency panel if the user tries to type into the disabled output currency panel
+                  onDisabledClick: () =>
+                    window.dispatchEvent(new Event(`forceInputFocus:${InterfaceSectionName.CURRENCY_INPUT_PANEL}`)),
```

Comment:
can we just remove this behavior? the input is disabled, and it appears disabled, so i don't think we should be hacking in this reaction. because the Event-sending across different components does feel hacky and easy to break later

Reply:
thx was hoping to get feedback on this pattern, zach said the same, I'm gonna use refs instead.